File: Integração Sistemas Heterogêneos/Cliente/src/tsi/ws/modelo/Turma.java
package tsi.ws.modelo;

import java.util.List;

public class Turma {
	private int id,anoTurma;
	private String curso,nomeTurma;
	private List<Aluno> alunos;
	private static int identificacaoTurma;
	public Turma() {
		id = ++identificacaoTurma;
	}
		
	public Turma(int anoTurma, String curso, String nomeTurma, List<Aluno> alunos) {
		super();
		this.anoTurma = anoTurma;
		this.curso = curso;
		this.nomeTurma = nomeTurma;
		this.alunos = alunos;
	}

	public String getNomeTurma() {
		return nomeTurma;
	}

	public void setNomeTurma(String nomeTurma) {
		this.nomeTurma = nomeTurma;
	}

	public List<Aluno> getAlunos() {
		return alunos;
	}

	public void setAlunos(List<Aluno> alunos) {
		this.alunos = alunos;
	}

	public void setId(int id) {
		this.id = id;
	}

	public int getId() {
		return id;
	}
	
	public int getAnoTurma() {
		return anoTurma;
	}
	public void setAnoTurma(int anoTurma) {
		this.anoTurma = anoTurma;
	}
	public String getCurso() {
		return curso;
	}
	public void setCurso(String curso) {
		this.curso = curso;
	}
		
}

File: Integração Sistemas Heterogêneos/Servidor/Turma.py
#Class

class Turma:
	def __init__(self, id, anoTurma, cursoTurma,nomeTurma):
		self.id = id
		self.anoTurma = anoTurma
		self.cursoTurma = cursoTurma
		self.nomeTurma = nomeTurma

	def getNomeTurma(self):
		return self.nomeTurma

	def getId(self):
		return self.id

	def getAnoTurma(self):
		return self.anoTurma

	def getSiglaTurma(self):
		return self.cursoTurma


	def setNomeTurma(self,nomeTurma):
		self.nomeTurma = nomeTurma

	def setIdTurma(self, idTurma):
		self.idTurma = idTurma

	def setAnoTurma(self, anoTurma):
		self.AnoTurma = anoTurma

	def setCursoTurma(self, cursoTurma):
		self.cursoTurma = cursoTurma


	#toString
	def __str__(self):
		return "A turma " + str(self.nomeTurma) + " de " + str(self.anoTurma) + " do curso " + str(self.cursoTurma)